src/test/java/com/bitso/bitsoLedgerEntryTest.json

```diff
@@ -0,0 +1,3 @@
+{"success":true,"payload":[]}
```

Comment:
+Moved the file with the json objects to a top-level folder. Not sure if also the test cases files have to be moved to the suggested folder, please confirm.